### PR TITLE
Update script location for daily CyHy notification cron job

### PR DESCRIPTION
**Pull request by dav3r**
The location of the daily notification script (`create_send_notifications.py`) changes in cisagov/ansible-role-cyhy-reports#4.  This PR updates the cron job to point to the new location.

This PR should be deployed in conjunction with cisagov/ansible-role-cyhy-reports#4 and cisagov/cyhy-mailer#54.

### CODE REVIEW

File: ansible/roles/cyhy_reporter/tasks/main.yml [hunk 1]
```diff
@@ -61,10 +61,10 @@
...
   cron:
     name: "Generate and send daily CyHy notifications"
-    minute: 0
-    hour: 10
+    minute: '0'
+    hour: '10'
     user: cyhy
-    job: cd /var/cyhy/notifications && ./create_send_notifications.py --log-level=info cyhy 2>&1 | /usr/bin/logger -t cyhy-notifications
+    job: cd /var/cyhy/reports && ./create_send_notifications.py --log-level=info cyhy 2>&1 | /usr/bin/logger -t cyhy-notifications
   when: production_workspace
```
Comment (jsf9k) [inline ansible/roles/cyhy_reporter/tasks/main.yml]: This should be `production_workspace|bool` to avoid an Ansible 2.8 warning.

**Review by jsf9k — CHANGES_REQUESTED** [review]
Requesting one minor change.  Otherwise this looks good to me.

File: ansible/roles/cyhy_reporter/tasks/main.yml (hunk 1 above)
Comment (dav3r) [reply ansible/roles/cyhy_reporter/tasks/main.yml]: Good catch, thanks @jsf9k - fixed in 2678ef5

**Review by jsf9k — APPROVED** [review]
Huzzah!

**Pull request merged**